src/style.css

```diff
@@ -1 +1,68 @@
 /* Styles go here */
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  min-width: 1024px;
+  max-width: 1200px;
```

Comment:
Instead of max-width, use media query to adjust layout for screens larger than 1200px.

Reply:
can't solved it